trial-config-service/src/main/java/uk/ac/ox/ndph/mts/trialconfigservice/config/SecurityConfig.java

```diff
@@ -0,0 +1,33 @@
+package uk.ac.ox.ndph.mts.trialconfigservice.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.http.HttpMethod;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.builders.WebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
+
+
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http.authorizeRequests()
+            .antMatchers(HttpMethod.OPTIONS,  "/trial-config/**").permitAll()
+            .anyRequest().authenticated()
+            .and().oauth2ResourceServer(oauth2 -> oauth2.jwt()
+                /*.jwtAuthenticationConverter(getCustomJwtAuthConverter())*/);// not yet
```

Comment:
Please add a TODO comment with a user story number for any technical debt or future work to be done.

Reply:
Added to tech debt